apko-build/action.yaml

```diff
@@ -69,7 +75,7 @@ inputs:
 
 runs:
   using: docker
-  image:  "docker://ghcr.io/wolfi-dev/apko@sha256:3f7077f5a9d74747c03c03e18d202e81343e654f1b33220220af76db914dc1d1"
+  image: "docker://${{ inputs.apko-image }}"
```

Comment:
looks like github doesnt like this to be used here

Reply:
yeah it seems that way:
```
Error: mattmoor/actions-2/bd30d83af1ef34bb40c16b379db7b6b814cf397b/apko-build/action.yaml (Line: 78, Col: 10):
Error: mattmoor/actions-2/bd30d83af1ef34bb40c16b379db7b6b814cf397b/apko-build/action.yaml (Line: 78, Col: 10): Unrecognized named-value: 'inputs'. Located at position 1 within expression: inputs.apko-image
Error: Fail to load mattmoor/actions-2/bd30d83af1ef34bb40c16b379db7b6b8[14](https://github.com/chainguard-images/images/actions/runs/4854919068/jobs/8652879224?pr=530#step:4:14)cf397b/apko-build/action.yaml
```

Reply:
I also missed passing this thru `apko-snapshot`.

I am going to try changing these `using: docker` steps to simply invoke `docker run` directly, and circle back.  I'll ping the PR when this is RFAL

Reply:
@jdolitsky this is RFAL, I pulled it in and got it working in images: https://github.com/chainguard-images/images/pull/530